app/types/realm-profile.ts

```diff
@@ -13,6 +13,9 @@ export interface RealmProfile {
   technical_contact_email: string;
   technical_contact_idir_userid: string;
   technical_contact_name: string;
+  second_technical_contact_email: string;
+  second_technical_contact_idir_userid: string;
+  second_technical_contact_name: string;
```

Comment:
actually we don't need `second_technical_contact_name` as it is dynamically added via Keycloak API calls, can you double-check how `technical_contact_name` has been populated at backend?